src/components/Relation.vue

```diff
@@ -512,6 +520,7 @@
             deferLoading:   data_from_server && this.table.count,
             ajax: data_from_server ? async (opts) => {
               try {
+                this.change_ordering = this.change_ordering || [{ column: 1, dir: 'asc' }].reduce((a, o, i) => a && (o.column !== opts.order[i].column || o.dir !== opts.order[i].dir), true);
```

Comment:
@volterra79 please add a comment, It's a bit unclear what is going here.

Reply:
@volterra79 I'm referring to this part 🧙: `[{ column: 1, dir: 'asc' }].reduce( ...`,

perhaps you meant?

```js
this.change_ordering = this.change_ordering || opts.order.every((o, i) => o.column !== 1 || o.dir !== 'asc');
```

Reply:
or even simpler:

```js
this.change_ordering = this.change_ordering || (1 !== opts.order[0].column || 'asc' !== opts.order[0].dir);
```

Reply:
@Raruto approved. Please change it in this way